README.md

```diff
@@ -156,11 +156,21 @@ which is useful for interacting with the Feed Registry.
 In general, these registries help you access essential contract addresses deployed on different networks,
 making it easier to integrate Chainlink services into your projects.
 
-You can access the plugin registries using one of the following methods:
+Below is a list of the registries provided by Hardhat Chainlink plugin:
```

Comment:
Perhaps rephrase to "Below is a list of the objects that contain address data for service configurations for the Chainlink services exposed by the Hardhat Chainlink Plugin"

or something along those lines